nd4j/nd4j-backends/nd4j-backend-impls/nd4j-cuda/src/main/java/org/nd4j/linalg/jcublas/buffer/BaseCudaDataBuffer.java

```diff
@@ -747,27 +763,32 @@ public void set(long[] data, long length, long srcOffset, long dstOffset) {
 
             }
             break;
+
+            case UBYTE: {
+                data = ArrayUtil.cutBelowZero(data);
+                for (int e = 0; e < data.length; e++) {
+                    put(e, data[e]);
+                }
+
+                syncDeviceAndHost();
+
+            }
```

Comment:
Is this explicitly meant to fall through to the next case? If so, put a note here.